README.md

```diff
@@ -31,23 +31,42 @@ print(response.json())
 ```
 
 #### JavaScript
-[Code example required](https://github.com/WISVCH/payments/issues/6)
+
+Creating an order:
+```js
+fetch('http://localhost:9000/api/orders', {
+  method: 'POST',
+  body: {
+    name: 'Thomas Ticket',
+    email: 'thomasticket@example.com',
+    returnUrl: 'https://www.ch.tudelft.nl/payments/ordercompleted',
+    productKeys: ['e44685c5-a360-467c-960d-29843a101bb1', '879941f4-43d1-4ff2-ad66-cbdf3b141cab']
+  }
+});
+```
+
+Obtaining the status:
+```js
+const request = await fetch('http://localhost:9000/api/orders/status/e44685c5-a360-467c-960d-29843a101bb1');
```

Comment:
I think this example should use the order id from the request you have done above